docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -218,7 +218,17 @@ To improve the Developer Experience when developing or using the administration
 
 ### Exceptions
 
-- `uid` — This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+**`uid`**
+
+This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
```

Comment:
`string` or `text`

Reply:
This line is from the original documentation, but I've made the correction